src/solve/mod.rs

```diff
@@ -180,12 +67,6 @@ impl fmt::Display for Solution {
 
 #[derive(Copy, Clone, Debug, PartialOrd, Ord, PartialEq, Eq, Hash)]
 pub enum SolverChoice {
```

Comment:
This is now a single-variant enum, which is a bit funky, but I left it the way it is since it actually reads pretty well IMO.

Reply:
Seems ok. We may add more options later or something. (Or we can always clean it up later.)